sdks/python/setup.py

```diff
@@ -184,7 +184,7 @@ def get_version():
     'cachetools>=3.1.0,<5',
     'google-apitools>=0.5.31,<0.5.32',
     'google-auth>=1.18.0,<2',
-    'google-cloud-datastore>=1.8.0,<2',
+    'google-cloud-datastore>=1.15.3,<2',
```

Comment:
To make it easier for others to depend on Beam, it's better to keep `python_requires` bounds as permissive as possible. I suggest to keep current bounds here.

Reply:
I see. I've changed it back.